pkg/cmd/repo/fork/fork.go

```diff
@@ -317,8 +324,28 @@ func forkRun(opts *ForkOptions) error {
 			}
 		}
 		if cloneDesired {
-			forkedRepoURL := ghrepo.FormatRemoteURL(forkedRepo, protocol)
-			cloneDir, err := gitClient.Clone(ctx, forkedRepoURL, opts.GitArgs)
+			// Allow injecting alternative BackOff in tests.
+			if opts.BackOff == nil {
+				bo := backoff.NewExponentialBackOff()
+				bo.Multiplier = 1.1
+				bo.MaxInterval = 10 * time.Second
+				bo.MaxElapsedTime = 5 * time.Minute
```

Comment:
I have absolutely no idea what proper numbers would be. I stole mine from something else because it was suggested.

IME, a clone very shortly after the failed clone works, but I don't have a great sense of scale and I'm not sure if it relates to the size of a repository (and if that's by commits, by number of files in the working directory, or some other measure).

My clones come from repositories of various sizes somewhat randomly/haphazardly...